src/Acl.php

```diff
@@ -104,26 +104,9 @@ public function applyRestrictions(Model $m): void
                     $m->getUserAction($action)->enabled = false;
                 }
             }
-
-            // add conditions on model
-            /* this will work in future when we will have json encoded condition structure stored in here
-            if ($rule['conditions']) {
-                $this->applyConditions($m, $rule['conditions']);
-            }
-            */
         }
     }
 
-    /**
-     * Apply conditions on model.
-     *
-     * @param mixed $conditions
-     */
-    public function applyConditions(Model $m, $conditions): void
```

Comment:
remove also the commented "can" method code below

then this PR is RTM from my side